oz-gsn-run-relayer.js

```diff
@@ -16,6 +16,7 @@ program
   )
   .option('--no-register', 'skip registration of the relayer process')
   .option('--no-devMode', 'turns off dev mode in relayer')
+  .option('--fee', "relay's per transaction fee")
```

Comment:
```suggestion
  .option('--fee <percentage>', "relayer's transaction fee (defaults to 70)")
```

Could you also add a note to the documentation of [runRelayer](https://github.com/OpenZeppelin/openzeppelin-gsn-helpers/blob/master/docs/modules/ROOT/pages/api.adoc#runrelayer) so this shows up on the [docs site](https://docs.openzeppelin.com/gsn-helpers/0.2/api#runrelayer)? Thanks!

Reply:
Sure @nventuro, done.